utils/fetch.ts

```
@@ -2,35 +1,63 @@
+class Https {
+  BASE_URL: string = 'http://127.0.0.1:4523/m1/5347033-5018095-default'
+  authToken: string = ''
+  constructor() {}
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)